javaparser-core-testing/pom.xml

```diff
@@ -103,9 +103,8 @@
     <dependencies>
         <dependency>
             <groupId>com.github.javaparser</groupId>
-            <artifactId>javaparser-core</artifactId>
-            <version>${project.version}</version>
-            <scope>test</scope>
+            <artifactId>javaparser-symbol-solver-core</artifactId>
+            <version>3.25.5</version>

```

Comment:
I don't understand this commit?

Reply:
Sorry for my oversight. I mistakenly pushed changes to the pom files into this branch. I understand the concerns raised and will correct these errors promptly. I'll then split my contributions and open a new PR as you've suggested. Thank you for pointing out the inconsistencies.